plan/validator.go

```diff
@@ -271,6 +271,16 @@ func (v *validator) checkAlterTableGrammar(stmt *ast.AlterTableStmt) {
 			default:
 				// Nothing to do now.
 			}
+		case ast.AlterTableOption:
+			for _, opt := range spec.Options {
+				switch opt.Tp {
```

Comment:
I think it's enough to use `if-else` here. We may remove it when we intend to support this type in the future. And if we need to handle other types in the future, then we can update it at that time.